Cubical/Algebra/CommRing/Instances/QuoInt.agda

```diff
@@ -0,0 +1,55 @@
+{-
+
+ℤ is a Commutative Ring (using QuoInt)
+
+-}
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.CommRing.Instances.QuoInt where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Algebra.CommRing
+open import Cubical.Data.Nat  using (ℕ ; zero ; suc)
+open import Cubical.Data.Bool using (not)
+open import Cubical.Data.Int.MoreInts.QuoInt
+  renaming (ℤ to ℤType ; _+_ to _+ℤ_; _·_ to _·ℤ_; -_ to -ℤ_)
+
+open CommRingStr
+
+-- The missing additive inversion
+
++-lInv' : ∀ n s → signed (not s) n +ℤ signed s n ≡ signed s zero
```

Comment:
I'm not sure, if it is necessary in this case, but certainly good to follow the naming conventions, which say ```+InvL'``` in this case.

Reply:
done